projects/aca-playwright-shared/src/api/sites-api.ts

```diff
@@ -109,4 +109,38 @@ export class SitesApi {
       }
     }
   }
+
+  async createSiteMembershipRequestForPerson(personId: string, siteId: string): Promise<SiteMembershipRequestEntry> {
+    const body = {
```

Comment:
could you please set type? It should be SiteMembershipRequestBodyCreate probably

Reply:
type was set in [aa865dc](https://github.com/Alfresco/alfresco-content-app/pull/3545/commits/aa865dcf243348da373c4560117ab99260fb3fe8)